google/cloud/forseti/notifier/notifiers/inventory_summary.py

```diff
@@ -129,24 +144,32 @@ def _send_email(self, summary_data, details_data):
 
         gsuite_dwd_status = self._get_gsuite_dwd_status(summary_data)
 
-        email_content = email_util.render_from_template(
+        email_content = email_connector.render_from_template(
             'inventory_summary.jinja',
             {'inventory_index_id': self.inventory_index_id,
              'timestamp': timestamp,
              'gsuite_dwd_status': gsuite_dwd_status,
              'summary_data': summary_data,
              'details_data': details_data})
 
+        if self.notifier_config.get('email_connector'):
+            sender = (
+                self.notifier_config.get('email_connector').get('sender'))
+            recipient = (
+                self.notifier_config.get('email_connector').get('recipient'))
+        # else block below is added for backward compatibility.
+        else:
+            sender = email_summary_config.get('sender')
+            recipient = email_summary_config.get('recipient')
         try:
-            email_util.send(
-                email_sender=email_summary_config.get('sender'),
-                email_recipient=email_summary_config.get('recipient'),
-                email_subject=email_subject,
-                email_content=email_content,
-                content_type='text/html')
+            email_connector.send(email_sender=sender,
+                                 email_recipient=recipient,
+                                 email_subject=email_subject,
+                                 email_content=email_content,
+                                 content_type='text/html')
             LOGGER.debug('Inventory summary sent successfully by email.')
         except util_errors.EmailSendError:
-            LOGGER.exception('Unable to send inventory summary email')
+            LOGGER.warn('Unable to send Violations email')
```

Comment:
Keep this at exception level so that the stacktrace is captured.

Reply:
Done